lib/awful/widget/common.lua

```diff
@@ -46,6 +46,8 @@ local function default_template()
     local ibm = wibox.container.margin(ib, dpi(4))
     local l   = wibox.layout.fixed.horizontal()
 
+    bgb:set_border_strategy("inner")
```

Comment:
Isn't that kind of a backwards-incompatible change in the default?

Reply:
Also, I just noticed: There is no such property yet. Either GitHub shows me the commits in a wrong order, or you use this strategy thing before it exists.

Reply:
> Isn't that kind of a backwards-incompatible change in the default?

Given you already proposed a change to the background `clip`, I see this has the logical follow up. Just as the `clip`, yes, it is a behavior change. Just as the clip, I think it falls into the "this is what everybody probably wants anyway" category?

> Either GitHub shows me the commits in a wrong order, or you use this strategy thing before it exists.

GitHub commits for updated PRs are not ordered.